### ISSUE TITLE
Calling PropTypes validators directly is not supported by the `prop-types` package

### ISSUE THREAD

**Issue by bitriddler** [styled-components/styled-components#1090]
## Versions
styled-components: 2.1.2
react: 15.6.1

I get this warning whenever I try to use a styled component
```
Warning: Failed Context Types: Calling PropTypes validators directly is not supported by the `prop-types` package. Use `PropTypes.checkPropTypes()` to call them. Read more at http://fb.me/use-check-prop-types
```

**Comment by BeijiYang** [comment]
I have the same problem when I try to style a third-party component (such as an antd Button) ,
but  " const Div = styled.div`` "works perfectly.

it drives me crazy.

**Comment by BeijiYang** [comment]
I did this:

```
rm -r node_modules
npm i
```

and this warning was gone.
